lm75bd/lm75bd.c

```diff
@@ -27,8 +30,19 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  uint8_t tmp_reg_pointer = LM75BD_TEMP_REG;
```

Comment:
variable names should be camelCase. This applies everywhere else for your other variables as well